src/main/configurations/Translate/DeploymentSpecifics.properties

```diff
@@ -96,6 +96,8 @@ ISO639_Codes_path=${configurations.directory}/Translate/ISO639.xml
 creeerZaak.timeout=20000
 
 Prefixparam=1900
+PrefixparamDate=1900ZK
+NewIdDigit=00000
```

Comment:
These variables are supposed to be calculated runtime. Prefixparam is no longer needed as it is part of the template string. The date should be gotten at runtime and should be current date/time the moment the new identifier is created.
NewIdDigit should simply be the new identifier with the formatting applied to it. So the {NewIdentificatie} variable used to update the database, can be used as input for the formatting xslt.

In the properties there should only be a template string like:
zaak-identificatie-template="ZK{datetime:yyyy}-{id:D5}"